docs/operational_guide/placement.md

```diff
@@ -0,0 +1,205 @@
+# Placement
+
+## Overview
+
+**Note**: The words *placement* and *topology* are used interchangeably throughout the M3DB documentation and codebase.
```

Comment:
not a 100% but i think `Topology` and `Placement` are proper nouns for the sake of this document - i.e. they should always start Topology not topology (in the whole document)

Reply:
spoke offline, gonna stick with lowercase